caramel/compiler/ocaml_to_erlang/names.ml

```diff
@@ -149,15 +149,19 @@ let ocaml_to_erlang_primitive_op t =
   match t with
   | "!" | "++" | "-" | "--" | "<" | ">" | "*" | "+" -> to_erl_op t
   | "/" -> to_erl_op "div"
+  | "mod" -> to_erl_op "rem"
   | "+." -> to_erl_op "+"
   | "-." -> to_erl_op "-"
   | "*." -> to_erl_op "*"
   | "/." -> to_erl_op "/"
+  | "float_of_int" -> to_erl_op "float"
```

Comment:
We should put these in the `external` syntax as follows:

```ocaml
(* in Math.ml *)
external float_of_int : int -> float = "float"
```

that way this names.ml module doesn't have to keep growing and we can keep it for edge-cases. Hopefully we'll get rid of most of it in the future and all operators would be defined as externals.

Reply:
@ostera I'm afraid I can't put them in `Math.ml` just yet because it would try to call `:math.float` which doesn't exist?

Reply:
I can move them to `Erlang.ml` for now.

Reply:
Oh, my bad, yes `Erlang.ml` sounds good! 🙌🏽